docs/product/integrations/source-code-mgmt/github/index.mdx

```diff
@@ -296,13 +296,13 @@ By default, this feature is automatically enabled once your GitHub integration h
 
 <Note>
 
-Sentry will automatically try to set up code mappings on Python, JavaScript, Node.js, and Ruby projects for organizations with the GitHub integration installed. If your project uses a different SDK, you can add code mappings manually. Support for other languages and other source code integrations is being worked on.
+Sentry will automatically try to set up code mappings on Python, JavaScript, Node.js, Ruby, and PHP projects for organizations with the GitHub integration installed. If your project uses a different SDK, you can add code mappings manually. Support for other languages and other source code integrations is being worked on.
 
 </Note>
 
 Stack trace linking takes you from a file in your Sentry stack trace to that same file in your source code. If you have commit tracking set up in Sentry, we can take you to the exact version (using the commit associated with the event) of the source code when the error occurred. Otherwise we'll link you to the current state of the source code (using the default **branch**).
 
-1. Navigate to **Settings > Integrations > GitHub > Configurations**.
+ > GitHub > Configurations**.
```

Comment:
bonus change??

Reply:
Nope must've fat finger this change, reverting now